rulesets/rules-termit-graphdb.pie

```diff
@@ -91,11 +91,38 @@ Rules
     -------------------------------
     this <termit:má-zdroj-definice-termu> source
 
+  Id: pdp_term_9
+
+      this <rdf:type> <skos:Concept>
+      this <skos:related> related
+      -------------------------------
+      related <skos:related> this
+
+  Id: pdp_term_10
+
+        this <rdf:type> <skos:Concept>
+        this <skos:relatedMatch> related
+        -------------------------------
+        related <skos:relatedMatch> this
+
   Id: sioc_comment_1
 
       source <rdf:type> <termit:reakce>
       source <activity:object> this
       -------------------------------
       this <termit:má-reakci> source
+
+  Id: skos_exact_match_symmetry
+
+      source <skos:exactMatch> this
+      -------------------------------
+      this <skos:exactMatch> source
+
+  Id: skos_exact_match_transitivity
+
+      x <skos:exactMatch>    y           [Constraint x != y, x != z, y != z]
+      y <skos:exactMatch>    z
+      -------------------------------
+      x <skos:exactMatch>    z
```

Comment:
Shouldn't we unify pdp_term_9, pdp_term_10 and skos_exact_match_symmetry and skos_exact_match_transitivity rules? E.g., the related-ones expect the subject/object to be a Concept while the exactMatch not (curiously, the corresponding SPIN rule does).

Reply:
skos:Concept is the domain of these relationships. I removed the skos:Concept check from the SPIN rule.